tests/cpp-tests/Classes/controller.cpp

```diff
@@ -73,15 +73,15 @@ class RootTests : public TestList
 //        addTest("Extensions", []() { return new ExtensionsTests(); });
         addTest("FileUtils", []() { return new FileUtilsTests(); });
         addTest("Fonts", []() { return new FontTests(); });
-        addTest("Interval", [](){return new IntervalTests(); });
+//        addTest("Interval", [](){return new IntervalTests(); }); // TODO crash on iPhone X
 #if (CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID)
         addTest("JNIHelper", []() { return new JNITests(); });
 #endif
 //        addTest("Material System", [](){return new MaterialSystemTest(); });
 //        addTest("Navigation Mesh", [](){return new NavMeshTests(); });
 //        addTest("Node: BillBoard Test", [](){  return new BillBoardTests(); });
 //        addTest("Node: Camera 3D Test", [](){  return new Camera3DTests(); });
-        addTest("Node: Clipping", []() { return new ClippingNodeTests(); });
+//        addTest("Node: Clipping", []() { return new ClippingNodeTests(); }); // TODO crash on iPhone X
```

Comment:
should comment all clipping test?

Reply:
2 items `TODO` existed at `CCClippingNode.cpp`, the first case don't crash on macOS, but iPhone X

Reply:
can just comment crashed test cases?

Reply:
sure, I will check. 

> when the first case crash, I think most of cases will crash.